### ISSUE TITLE
Audio sent to gst-switch sounds terrible

### ISSUE THREAD

**Issue by CarlFK** [timvideos/gst-switch#68]
gst-launch-1.0 audiotestsrc ! audioconvert ! alsasink
plays a tone, sounds fine.

start server, start ui
gst-launch-1.0     audiotestsrc    ! queue     ! audioconvert    ! gdppay     ! tcpclientsink port=4000

sounds terrible.  the visualization looks like what I would expect from terrible sound.  
the recording has the same terrible sound.

**Comment by mithro** [comment]
Should be fixed in https://github.com/timvideos/gst-switch/pull/99

**Comment by mithro** [comment]
Fixed as #99 is merged.

Command needed for good audio is;
```
gst-launch-1.0 audiotestsrc is-live=true \
        ! audioconvert \
        ! audio/x-raw,rate=48000,channels=2,format=S16LE,layout=interleaved \
        ! gdppay \
        ! tcpclientsink port=4000
```

That pipeline is in the README.md now too.
